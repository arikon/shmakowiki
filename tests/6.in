para1**bla**
//dsds//

para2
para2
