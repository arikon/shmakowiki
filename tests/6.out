[[para, [para1, [bold, [bla]], 
, [italic, [dsds]]]], [para, [para2
para2]]]
